clients/src/main/resources/common/message/DeleteShareGroupStateRequest.json

```diff
@@ -0,0 +1,31 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "apiKey": 80,
+  "type": "request",
+  "listeners": ["broker"],
+  "name": "DeleteShareGroupStateRequest",
+  "validVersions": "0",
+  "flexibleVersions": "0+",
+  "fields": [
+    {"name": "GroupId", "type": "string", "versions": "0",
```

Comment:
I notice the KIP uses versions "0" when they should all be "0+". I'll fix the KIP when I respond to your comment in the thread. Please change these to "0+" in all cases. Thanks.